docs/src/main/sphinx/connector/hive.rst

```diff
@@ -984,6 +984,85 @@ Procedures
   Flush Hive metadata cache entries connected with selected partition.
   Procedure requires named parameters to be passed
 
+Table properties
+----------------
+
+.. list-table:: Hive connector storage table properties
+  :widths: 20, 60, 20
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default
+  * - ``auto_purge``
+    - The skip trash when table or partition is deleted.
+    -
+  * - ``avro_schema_url``
+    - The ``URI`` pointing to Avro schema for the table.
+    -
+  * - ``bucket_count``
+    - The number of buckets to group data into. 
+      Must be used with ``bucketed_by``.
+    -
+  * - ``bucketed_by``
+    - The bucketing column for the storage table. 
+      Must be used with ``bucketed_count``.
+    -
+  * - ``bucketing_version``
+    - The bucketing version.
+    -
+  * - ``csv_escape``
+    - CSV escape character.
+    -
+  * - ``csv_quote``
+    - The ``CSV`` quote character.
+    -
+  * - ``csv_separator``
+    - The ``CSV`` separator character.
+    -
+  * - ``external_location``
+    - The file system, URI location for an external Hive 
+      table on S3, Azure Blob Storage, etc.
+    -
+  * - ``format``
+    - The file format to use in the storage. Valid values include ``ORC``,
+      ``PARQUET``, ``CSV``, ``JSON`` and others. The catalog property
+      ``hive.storage-format`` sets the default value to ``ORC`` and can be used
+      to set a different default.
+    - ``ORC``
+  * - ``null_format``
+    - The serialization format for ``NULL`` value.
```

Comment:
`Can only be used with ``TEXTFILE``, ``RCTEXT`` or ``SEQUENCEFILE`` format tables.`